diff -up a/config.def.h b/config.def.h
--- a/config.def.h	2023-09-09 16:25:42.652453824 +0200
+++ b/config.def.h	2023-09-09 16:24:12.820238713 +0200
@@ -107,7 +107,7 @@ static Key keys[] = {
 	/* needs spmenu */
 	{ Mod4Mask,			XK_v,	   spawn,	   SHCMD("clipmenu-spmenu") },
 	{ Mod4Mask|ShiftMask,		XK_Print,  spawn,	   SHCMD("screenshot-spmenu -f") },
-	{ Mod4Mask|Mod1Mask,		XK_Print,  spawn,	   SHCMD("screenshot-spmenu -s") },
+	{ Mod4Mask|Mod1Mask,		XK_Print,  spawn,	   SHCMD("screenshot-spmenu -s") },
 	{ Mod4Mask,			XK_p,	   spawn,	   SHCMD("pirokit") },
 	{ Mod4Mask|ShiftMask,		XK_w,	   spawn,	   SHCMD("wallpaper-spmenu") },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          SHCMD("st -e") },
@@ -141,8 +141,7 @@ static Key keys[] = {
 	{ MODKEY|ShiftMask,             XK_0,      tag,            {.ui = ~0 } },
 	{ MODKEY,                       XK_comma,  focusmon,       {.i = -1 } },
 	{ MODKEY,                       XK_period, focusmon,       {.i = +1 } },
-	{ ControlMask|ShiftMask,        XK_s,      show,           {0} },
-	{ ControlMask|ShiftMask,        XK_h,      hide,           {0} },
+	{ ControlMask|ShiftMask,        XK_h,      togglewin,      {0} },
 	{ MODKEY|ShiftMask,             XK_comma,  tagmon,         {.i = -1 } },
 	{ MODKEY|ShiftMask,             XK_period, tagmon,         {.i = +1 } },
 	{ MODKEY|ShiftMask,             XK_h,      layoutscroll,   {.i = -1 } },
